pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "schnax"
+version = "0.1.0"
+description = "schnet in jax"
+authors = ["Fabian Nagel <fabian.2.nagel@uni-konstanz.de>", "Marcel Langer <dev@marcel.science>"]
+
+[tool.poetry.dependencies]
```

Comment:
do we really want go all-in on bleeding edge dependencies from master branches? I'm almost sure that will mysteriously break at some point 😅

Reply:
This is mostly for convenience while co-developing the other tools. `ase` has continuous integration so shouldn't break on the fly.

Reply:
(I don't mind making `ase` a normal dependency)